package.json

```diff
@@ -18,26 +18,26 @@
     "babel-runtime": ""
   },
   "devDependencies": {
-    "babel-core": "",
-    "babel-loader": "",
-    "babel-plugin-transform-decorators": "",
-    "babel-plugin-transform-decorators-legacy": "",
-    "babel-plugin-transform-runtime": "",
-    "babel-preset-es2015": "",
-    "babel-preset-stage-0": "",
-    "babel-preset-stage-1": "",
-    "babel-preset-stage-2": "",
-    "babel-preset-stage-3": "",
-    "babel-register": "",
+    "babel-core": "^6.22.1",
```

Comment:
When moving from shrinkwrap to package-lock NPM will try to update if the versions are not pinned. I've pinned to the versions that we're used previously in srhinkwrap using `^`, there are breaking changes in `7` which is picked otherwise. I'll address updating some of these dependencies separately. I think some can be removed.